cmd/notary/tuf.go

```diff
@@ -889,7 +889,13 @@ func tokenAuth(trustServerURL string, baseTransport *http.Transport, gun data.GU
 	if endpoint.Scheme == "" {
 		return nil, fmt.Errorf("Trust server url has to be in the form of http(s)://URL:PORT. Got: %s", trustServerURL)
 	}
-	subPath, err := url.Parse("v2/")
+	var p string
+	if strings.HasSuffix(endpoint.Path, "/") {
```

Comment:
Well... I was thinking we could detect the trailing slash and add if necessary. I don't feel strongly though, what you have currently is fine.